test/unit/module/core/test_run_cli.py

```diff
@@ -37,7 +37,7 @@ def test_template_not_found(self):
             ['--template', filename, '--ignore-bad-template'])
         (_, _, matches) = cfnlint.core.get_template_rules(filenames[0], args)
 
-        self.assertEqual(len(matches), 1)
+        self.assertEqual(len(matches), 0)
```

Comment:
This does change some output values.  Now ignore bad template will always return back 0 if the template fails in decode.